clients/spark/core/src/main/scala/io/treeverse/gc/DataLister.scala

```diff
@@ -60,7 +58,9 @@ class ParallelDataLister extends DataLister with Serializable {
       .map(_.path)
       .toDF("slice_id")
       .withColumn("udf", explode(objectsUDF(col("slice_id"))))
-      .withColumn("address", concat(col("slice_id"), lit("/"), col("udf._1")))
+      .withColumn("address",
```

Comment:
Suggestion: simply make this a column of "basename", and construct the address column outside this function.

Reply:
Done